<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog4</class>
 <widget class="QDialog" name="Dialog4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>ÖDÜNÇ TESLİM ALMA İŞLEMLERİ</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="GroupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>371</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTableWidget" name="txtKitapOduncDurumu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>361</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>Uye_no</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Kitap_No</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Odunc_Alma_Tarihi</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ödünç Alınan Kitapların Listesi</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="GroupBox_3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>80</y>
     <width>331</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTableWidget" name="txtKitapGecmisOduncDurumu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>331</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>Uye_No</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Kitap_No</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>kitap_sayisi</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Alma_Tarihi</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Borc</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>50</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Teslim Edilen Kitaplar Listesi</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtUyeNo">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(179, 179, 179);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtKitapNo">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>220</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(179, 179, 179);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnOduncuVer">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>340</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ödüncü Ver</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>290</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Uye_No :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>190</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Kitap_No :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>250</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ödünç Verme Tarihi :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
